diff --git loris/webapp.py loris/webapp.py
index ab5818b..8306029 100755
--- loris/webapp.py
+++ loris/webapp.py
@@ -452,7 +452,7 @@ class Loris(object):
         return r
 
     def get_favicon(self, request):
-        f = path.join(self.www_dp, 'icons', 'loris-icon.png')
+        f = path.join(self.www_dp, 'icons', 'favicon.ico')
         r = Response(file(f), content_type='image/x-icon')
         if self.enable_caching:
             r.add_etag()
@@ -700,9 +700,9 @@ if __name__ == '__main__':
 
     sys.path.append(path.join(project_dp)) # to find any local resolvers
 
-    app = create_app(debug=True) # or 'opj'
+    app = create_app(debug=False, config_file_path=path.join(project_dp, 'etc', 'loris2.conf')) # or 'opj'
 
-    run_simple('localhost', 5004, app, use_debugger=True, use_reloader=True)
+    run_simple('0.0.0.0', 5004, app, use_debugger=False, use_reloader=True)
     # To debug ssl:
     # run_simple('localhost', 5004, app, use_debugger=True, use_reloader=True,
     #     ssl_context='adhoc')
